lib/utils/tracker.js

```diff
@@ -18,20 +22,36 @@ class Tracker {
     ga.setTrackerId(trackerId);
   }
 
+  _isEnabled() {
+    const value = store.getState() ?
+      store.getState().userFlags.enableTelemetry.value :
+      TELEMETRY_DEFAULT_ENABLED;
+    return value;
+  }
+
   view(name) {
     debug('view', name);
```

Comment:
Yes, the debug message won't do anything on a production build, and want to leave it there even if the telemetry is off